src/include/sof/audio/kpb.h

```diff
@@ -58,6 +58,11 @@ struct comp_buffer;
 #define KPB_BYTES_TO_S16_SAMPLES(s)	((s) >> 1)
 #define KPB_BYTES_TO_S32_SAMPLES(s)	((s) >> 2)
 
+/* Macro that returns mask with selected bit set */
+#define COUNT_TO_BITMASK(cnt) (((0x1 << (cnt)) - 1))
+#define IS_BIT_SET(value, idx) ((value) & (1 << (idx)))
```

Comment:
`BIT(idx)`

Reply:
Also need to prefix new KPB macros with ```KPB_``` to avoid collisions with other header files.

Reply:
I have added the KPB_